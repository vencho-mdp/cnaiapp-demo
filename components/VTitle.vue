<template>
  <h1 class="font-bold text-lg"><slot /></h1>
</template>

<script>
export default {
  name: "Title",
};
</script>

<style></style>
