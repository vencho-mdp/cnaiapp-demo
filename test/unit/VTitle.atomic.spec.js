import { mount } from '@vue/test-utils'
import VTitle from '@/components/VTitle.vue'

describe('VTitle', () => {
  it('should render correctly', async () => {
    const el = mount(VTitle, {
      slots: {
        default: 'Title example'
      }
    })
    const text = el.find('h1').text()
    const html = el.html()
    expect(text).toBe('Title example')
    expect(html).toMatchSnapshot()
  })
})
