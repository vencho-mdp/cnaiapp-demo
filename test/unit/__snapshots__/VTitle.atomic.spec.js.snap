// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VTitle should render correctly 1`] = `"<h1 class=\\"font-bold text-lg\\">Title example</h1>"`;
